src/config/print/config.md

```diff
@@ -30,15 +17,31 @@ configure a printer using the command line.
 
 ## Graphical interface
 
-To configure the printer using a native GUI interface, run the following:
+The `system-config-printer` package offers simple and robust configuration of
+new printers. Install the package:
 
 ```
 # xbps-install -S system-config-printer
-# system-config-printer
 ```
 
-This command does not need to be run as root if PolicyKit is in use and
-`cups-pk-helper` is installed.
+Normally this tool would require root privileges to configure printers, but
+unprivileged access through `polkit` can be achieved with a helper. If you are
+interested in this, install the helper:
```

Comment:
if you are using PolicyKit, you can install the `cups-pk-helper` package to allow unprevileged users access to CUPS configuration.

Reply:
Well one note I would have here is that the CLI tools and web interface would still need root. Just system-config-printer and some other GUI tools take advantage of the policykit helpers.